docs/api/covidcast-signals/nchs-mortality.md

```diff
@@ -0,0 +1,56 @@
+| `influenza_deaths_prop` | Number of weekly new deaths involving Influenza per 100,000 population |
+| `pneumonia_deaths_num` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths |
+| `pneumonia_deaths_prop` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths per 100,000 population |
+| `pneumonia_and_covid_deaths_num`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza |
+|`pneumonia_and_covid_deaths_prop`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza per 100,000 population |
+|`pneumonia_influenza_or_covid_19_deaths_num`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19|
+|`pneumonia_influenza_or_covid_19_deaths_prop`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19 per 100,000 population|
+|`percent_of_expected_deaths`| The number of weekly new deaths for all causes in 2020 compared to the average number across the same week in 2017–2019|
+
+These signals are taken directly from [Table 1](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm) without changes. National provisional counts include deaths occurring within the 50 states and the District of Columbia that have been received and coded as of the date specified. The deaths are classified based on a new ICD-10 code. The codes that are considered for each signals are described in details [here](https://github.com/cmu-delphi/covidcast-indicators/blob/nchs_mortality/nchs_mortality/DETAILS.md#metrics-level-1-m1). We export the state-level data as-is in a weekly format. 
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Geographical Exceptions
+
+New York City is considered as a special state in the NCHS Mortality data, but we don't consider NYC separately. The death counts for NYC would be included in New York State in our reports.
+
+## Report Using Epiweeks
+
+We report the NCHS Mortality data in a weekly format (weekly_YYYYWW, where YYYYWW refers to an epiweek). However, NCHS reports their weekly data from Saturday to Saturday. We assume there is a one day shift. For example, they report a death counts for Alaska in a week starting from date D, we will report the timestamp of this report as the corresponding epiweek of date(D + 1).
+
+## Data Versioning
+Data versions are tracked on both a daily and weekly level. On a daily level, we check for updates for NCHS mortality data every weekday as how it is reported by CDC and stash these daily updates on S3, but not our API. On a weekly level (on Mondays), we additionally upload the changes to the data made over the past week (due to backfill) to our API.
+
+## Lag and Backfill
+There is a lag in time between the death occurred and when the death certificate is completed, submitted to NCHS and processed for reporting purposes. This delay can range from 1 week to 8 weeks or even more, as described by NCHS. 
```

Comment:
This is the first time death certificates are mentioned. If these reports are based on death certificates, perhaps we should state that right in the opening sentence, so users understand what kind of data this is?

Also, this paragraph should go on to say that when NCHS backfills data, we issue new numbers in our API, etc.